### PR TITLE
Backport community edition gradle task

### PR DESCRIPTION
**Pull request by mkleen**
## Checklist

 - [ ] User relevant changes are recorded in ``CHANGES.txt``
 - [ ] Touched code is covered by tests
 - [X] Documentation has been updated if necessary
 - [X] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -330,22 +332,61 @@ task chooseReleaseNotes(dependsOn: [':common:getVersion']) {
 distTar {
     dependsOn ':common:getVersion'
     compression = Compression.GZIP
-    baseName = 'crate'
     extension = 'tar.gz'
 }
-distTar.dependsOn(chooseReleaseNotes)
```
Comment (seut) [inline app/build.gradle]: this differs from the original commit https://github.com/crate/crate/pull/8319/files#diff-39e7d8c00954e920b98e7636f0ac30b2R330. why that?

Comment (mkleen) [reply app/build.gradle]: Because there was a conflict, therefore mergify could not do it automatically. I cherry picked it and merged it manually.

Comment (seut) [reply app/build.gradle]: I was more interested about why the logic changed, like why not use `tasks.withType(Tar){..}` here as well?

Comment (mkleen) [reply app/build.gradle]: oh i see, you are right, this is wrongly merged. thanks!

Comment (mkleen) [reply app/build.gradle]: It should be merged correctly now.

**Pull request merged**